frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,303 @@
+                </div>
+
+                <div className="flex-none w-[220px]">
+                    <StatsCard
+                        title="Overall Completion"
+                        number="50"
+                        label="%"
+                        tooltip="Placeholder data"
+                    />
+                </div>
+
+                <div className="flex-1 card p-2 flex flex-col">
+                    <div className="flex items-center justify-between px-2">
+                        <h3 className="text-teal-4 uppercase">
+                            RATE OF COMPLETION
+                        </h3>
+                        <span className="uppercase text-teal-4">
+                            Past 30 days
+                        </span>
+                    </div>
+                    <ResponsiveContainer width="100%" height={120}>
+                        <LineChart data={completionRateData}>
+                            <CartesianGrid strokeDasharray="3 3" />
+                            <XAxis dataKey="name" /> <YAxis domain={[0, 100]} />
+                            <Tooltip labelStyle={{ color: '#000' }} />
+                            <Line
+                                type="monotone"
+                                dataKey="rate"
+                                stroke="var(--teal-3)"
+                                strokeWidth={2}
+                                name="Completion Rate"
+                            />
+                        </LineChart>
+                    </ResponsiveContainer>
+                </div>
+            </div>
+
+            <div className="flex flex-wrap items-center justify-between gap-2 mb-4">
+                <div className="flex flex-row gap-x-2">
+                    <SearchBar
+                        searchTerm={searchTerm}
+                        changeCallback={handleChange}
+                    />
+                    <DropdownControl
+                        label="order by"
+                        setState={setSortQuery}
+                        enumType={{
+                            'Name (A-Z)': 'name_last asc',
+                            'Name (Z-A)': 'name_last desc',
+                            'Section Created (Newest)': 'created_at desc',
+                            'Section Created (Oldest)': 'created_at asc'
+                        }}
+                    />
+                    {/* pill  */}
+                    <button
+                        className={`px-4 py-2 rounded-full border ${
+                            includeActive
+                                ? 'bg-teal-600 text-white'
+                                : 'bg-white text-teal-600'
+                        }`}
+                        onClick={() => setIncludeActive(!includeActive)}
+                    >
+                        {includeActive
+                            ? 'Active Sessions'
+                            : 'Inactive Sessions'}
+                    </button>
```

Comment:
After looking at this it feels off to me as a pill and almost looks like a button. Can we turn this into a checkbox that is unchecked by default? Checking it removes the filter and includes all sections. 

- [ ] Show All Sections